packages/queued-request-controller/package.json

```diff
@@ -66,7 +66,7 @@
   },
   "peerDependencies": {
     "@metamask/network-controller": "^18.0.0",
-    "@metamask/selected-network-controller": "^12.0.0"
+    "@metamask/selected-network-controller": "^13.0.0"
```

Comment:
Ah darn, this is a breaking change. We should release this package as well

Reply:
pushed here: https://github.com/MetaMask/core/pull/4260/commits/1bf4bf6873597e31357ff153a3ae5f5a3c0d5953